utils/constants.js

```diff
@@ -3,6 +3,7 @@ export const SERVER_URL =
   process.env.NEXT_PUBLIC_VERCEL_URL ||
   'http://localhost:3000';
 
+export const dockUrl = "https://api-testnet.dock.io";
```

Comment:
should this be referencing the env var DOCK_API_URL?

Reply:
Yeah thats right. Fixed.